tests/network-ovn

```diff
@@ -1051,6 +1051,10 @@ EOF
     dig a +tcp @192.0.2.1 u1.lxd
     dig aaaa +tcp @2001:db8:1:2::1 u1.lxd
 
+    echo "==> Check accessing load balancer's listen address inside the u1"
+    lxc exec u1 -- dig a +tcp @192.0.2.1 u1.lxd
```

Comment:
If I understand the test right, we should not be looking up the instance FQDN but `lxd.localdomain` that is the synthetic record that `dnsmasq` should return.

Ideally, we'd do a `dig +short` and compare the output with the expected `127.0.0.X`.

I know you just copied what was done before but I fear the risk of accidentally poking at LXD/OVN default DNS responder.